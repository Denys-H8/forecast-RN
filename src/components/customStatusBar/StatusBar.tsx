import React from 'react';
import { View, StatusBar } from 'react-native';

import constants from 'src/constants';
import { styles } from './statusBar.styles';

const CustomStatusBar: React.FC = () => (
  <View style={styles.statusBar}>
    <StatusBar barStyle="light-content" translucent backgroundColor={constants.BAR_BG_COLOR} />
  </View>
);

export default CustomStatusBar;
