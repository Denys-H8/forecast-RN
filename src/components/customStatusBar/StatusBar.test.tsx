import React from 'react';
import { render } from '@testing-library/react-native';
import StatusBar from './StatusBar';

test('StatusBar snapshot', () => {
  const { toJSON } = render(<StatusBar />);
  expect(toJSON()).toMatchSnapshot();
});
