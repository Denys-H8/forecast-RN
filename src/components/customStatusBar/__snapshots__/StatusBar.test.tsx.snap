// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusBar snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#191C39",
      "height": 20,
    }
  }
/>
`;
